stingray/simulator/simulator.py

```diff
@@ -28,39 +30,46 @@ class Simulator(object):
     rms : float, default 1
         fractional rms of the simulated light curve,
         actual rms is calculated by mean*rms
+    err : float, default 0
+        the errorbars on the final light curve bins
     red_noise : int, default 1
         multiple of real length of light curve, by
         which to simulate, to avoid red noise leakage
     random_state : int, default None
         seed value for random processes
+    poisson : bool, default False
+        return Poisson-distributed light curves.
     """
 
-    def __init__(self, dt, N, mean, rms, red_noise=1,
-                 random_state=None, tstart=0.0):
+    def __init__(self, dt, N, mean, rms, err=0., red_noise=1,
+                 random_state=None, tstart=0.0, poisson=False):
         self.dt = dt
- 
+
         if not isinstance(N, (int, np.integer)):
-            raise ValueError("N must be integer!") 
- 
+            raise ValueError("N must be integer!")
+
         self.N = N
 
         if mean == 0:
-            warnings.warn("Careful! A mean of zero is unphysical!" + \
+            warnings.warn("Careful! A mean of zero is unphysical!" +
                           "This may have unintended consequences!")
         self.mean = mean
         self.nphot = self.mean * self.N
         self.rms = rms
         self.red_noise = red_noise
         self.tstart = tstart
         self.time = dt*np.arange(N) + self.tstart
+        self.nphot_factor = 1000_000
```

Comment:
Is there supposed to be a `_` in ` 1000_000`?

Reply:
These are valid Python numbers, it's a cool way to distinguish 1000000 from 100000 :)